pkg/infra/grn/grn.go

```diff
@@ -0,0 +1,69 @@
+package grn
+
+import (
+	"fmt"
+	"strings"
+)
+
+type GRN struct {
+	// TenantID is specific to hosted grafana and will be omitted in other
+	// environments.
+	TenantID string
+
+	// OrgID contains the ID of the organization the resource belongs to. This
+	// field may be omitted for global Grafana resources which are not
+	// associated with an organization.
+	OrgID string
```

Comment:
@bergquist -- thoughts on Tenant vs Org?  and string vs number?

I think we need one or the other, not both.  I don't actually care which one.  But IIRC, orgs have a different meaning in cloud, so `TenantID` is likely a better choice

Reply:
Not a super strong opinion but I think tenantID is better long term. We can keep orgs semantic in the UI/docs etc and keep moving away from it in the code base until we can deprecate it. (at least the word).

single/multi tenancy is a known word in engineer while orgs is kinda vague.